src/autotype/AutoTypeSelectDialog.cpp

```diff
@@ -82,20 +83,20 @@ void AutoTypeSelectDialog::done(int r)
     QDialog::done(r);
 }
 
```

Comment:
This file needs a `keyPressEvent()` method to catch the return/enter key on macOS and emit the activated signal:

![image](https://user-images.githubusercontent.com/107672/35754633-3e500ffc-084b-11e8-8448-20d3ccfa7961.png)

Reply:
Strangely enought, this is already implemented in the EntryView class 
https://github.com/keepassxreboot/keepassxc/blob/c4b9d5408501a79cee0815d616d79d5a8f70815a/src/gui/entry/EntryView.cpp#L115

Reply:
>Edit: Now I've figure out why it worked before. Now the signal it's called emitMatchActivated and the EntryView is emitting the emitEntryActivated one :D

@TheZ3ro nice!